lib/action_controller/serialization.rb

```diff
@@ -8,8 +8,10 @@ module Serialization
     include ActionController::Renderers
 
     module ClassMethods
-      def serialization_scope(scope)
-        self._serialization_scope = scope
+      def serialization_scope(scope, opts = {})
+        before_action(opts.slice(:only, :except, :if, :unless)) do
```

Comment:
Since the method is only called at `render`, if not passed in as an option, I don't think I like having it eagerly evaluated in code that is pretty easy to handle in userland.  Would you be okay with removing the code changes in favor documenting using a `before_action` as an alternative to passing an option to `render`?